README.md

```diff
@@ -6,18 +6,17 @@
 
 Setting up a Local dev for your stack can be hard. You have to follow a lot of Blogs and do a lot of copying and pasting.  
 
-## Installation
+## Installation and Usage
 
 ```bash
 git clone https://github.com/nerdeveloper/stacks.git
 cd stacks
+./setup.sh
```

Comment:
This might give some issues with permission[chmod] but I am going to merge this and update the README.